src/main/scala/cromwell/engine/backend/jes/Run.scala

```diff
@@ -75,7 +75,7 @@ object Run  {
 
       // Update the database state:
       val newBackendInfo = JesCallBackendInfo(db.CallStatus(ExecutionStatus.Running.toString), Option(JesId(run.jesId)), Option(JesStatus(currentStatus.toString)))
-      dataAccess.updateExecutionBackendInfo(run.workflowId, run.call, newBackendInfo)
+      globalDataAccess.updateExecutionBackendInfo(run.workflowId, run.call, newBackendInfo)
```

Comment:
Please see DSDEEPB-1462, this is writing backend info without taking index into account.